fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    /* your code here */
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(r => r.json())
+      .then(r => {
+        const array = [];
+        r.forEach(e => { array.push(e); });
```

Comment:
We don't really need to do this, your `r` parameter is already an array of users